packages/math/src/pool/stable.ts

```diff
@@ -1,29 +1,38 @@
 import { Coin, Dec, DecUtils } from "@keplr-wallet/unit";
+import { BigDec } from "../big-dec";
 
 export const StableSwapMath = {
   calcOutGivenIn,
   calcInGivenOut,
   calcSpotPrice,
   // secondary
   solveCfmm,
-  compare_checkMultErrorTolerance,
+  compareDec_checkMultErrorTolerance,
```

Comment:
We should only add these if they need to be exposed to users of the package (the web package, currently), not just tests